scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -118,63 +124,98 @@ workflow GvsCreateFilterSet {
-      recalibration_tranche_values = ["100.0", "99.95", "99.9", "99.5", "99.0", "97.0", "96.0", "95.0", "94.0", "93.5", "93.0", "92.0", "91.0", "90.0"],
-      recalibration_annotation_values = indel_recalibration_annotation_values,
-      mills_resource_vcf = mills_resource_vcf,
-      mills_resource_vcf_index = mills_resource_vcf_index,
-      axiomPoly_resource_vcf = axiomPoly_resource_vcf,
-      axiomPoly_resource_vcf_index = axiomPoly_resource_vcf_index,
-      dbsnp_resource_vcf = dbsnp_vcf,
-      dbsnp_resource_vcf_index = dbsnp_vcf_index,
-      use_allele_specific_annotations = true,
-      disk_size = "1000",
-      machine_mem_gb = INDEL_VQSR_mem_gb_override,
-      max_gaussians = INDEL_VQSR_max_gaussians_override,
-      maximum_training_variants = INDEL_VQSR_maximum_training_variants,
+  # From this point, the paths diverge depending on whether they're using classic VQSR or VQSR-Lite
+  # The first branch here is VQSR-Lite, and the second is classic VQSR
+  if (!use_classic_VQSR) {
+    call VQSRLite.JointVcfFiltering as JointVcfFiltering {
+      input:
+        vcf = ExtractFilterTask.output_vcf,
+        vcf_index = ExtractFilterTask.output_vcf_index,
+        sites_only_vcf = MergeVCFs.output_vcf,
+        sites_only_vcf_index = MergeVCFs.output_vcf_index,
+        basename = filter_set_name,
+        gatk_docker = "us.gcr.io/broad-gatk/gatk:4.3.0.0",
+        extract_interval_list = interval_list,
+        score_interval_list = interval_list,
+        snp_annotations = "-A AS_QD -A AS_MQRankSum -A AS_ReadPosRankSum -A AS_FS -A AS_MQ -A AS_SOR",
+        indel_annotations = "-A AS_FS -A AS_ReadPosRankSum -A AS_MQRankSum -A AS_QD -A AS_SOR",
+        use_allele_specific_annotations = true,
+    }
+
+    call Utils.MergeVCFs as MergeINDELScoredVCFs {
+      input:
+        input_vcfs = JointVcfFiltering.indels_variant_scored_vcf,
+        gather_type = "CONVENTIONAL",
+        output_vcf_name = "${filter_set_name}.indel.vrecalibration.gz",
+        preemptible_tries = 3,
+    }
+
+    call Utils.MergeVCFs as MergeSNPScoredVCFs {
+      input:
+        input_vcfs = JointVcfFiltering.snps_variant_scored_vcf,
+        gather_type = "CONVENTIONAL",
+        output_vcf_name = "${filter_set_name}.snp.vrecalibration.gz",
+        preemptible_tries = 3,
+    }
+
+    call PopulateFilterSetInfo {
+      input:
+        gatk_override = gatk_override,
+        filter_set_name = filter_set_name,
+        snp_recal_file = MergeSNPScoredVCFs.output_vcf,
+        snp_recal_file_index = MergeSNPScoredVCFs.output_vcf_index,
+        indel_recal_file = MergeINDELScoredVCFs.output_vcf,
+        indel_recal_file_index = MergeINDELScoredVCFs.output_vcf_index,
+        fq_info_destination_table = fq_info_destination_table_vqsr_lite,
+        filter_schema = fq_info_destination_table_vqsr_lite_schema,
+        query_project = project_id,
+        useClassic = false
+    }
+
+    call PopulateFilterSetSites {
```

Comment:
I *think* this is a duplicate call in both code paths? Could go outside the loop?

Reply:
vqslod is just not a property passed out by vqsr_lite, but we didn't remove the column yet.   We're getting calibration_sensitivity instead, which gives the same information in a way that can be used much more directly to filter.  That's a deliberate difference in the design of VQSR vs VQSR-Lite.  Eventually, when we do our final cleanup pass, I think we should carefully look at which columns are no longer passed out and remove any that are vestigial.

Reply:
> I _think_ this is a duplicate call in both code paths? Could go outside the loop?

Hmm.  Two things, having talked it over in mobbing.
1, It does appear as though PopulateFilterSetSites, PopulateFilterSetInfo, and PopulateFilterSetTranches are independent of each other and can, in fact, be reordered.  I wasn't sure if that was possible, but it seems to be.
2. The inputs are currently coming from slightly different places inasmuch as the earlier tasks in the pipeline had to be given different names for the WDL to remain valid.  So they can't trivially be merge outside of the branch, without figuring out how to get them the correct inputs regardless of the branch we went through.

I'll leave it as is for now, and if/when we agree on VQSR-Lite and modify this file to use the latest version we can see if there's the possibility for convergence then.